iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -415,6 +417,8 @@ private static async void PingServer(object ctx)
                 {
                     // We've been idle for too long, send a ping!
                     await WriteMessageAsync(context, PingReqPacket.Instance, ShutdownOnWriteErrorHandler).ConfigureAwait(true);
+
+                    if (Logging.IsEnabled) Logging.Info(context, $"Idle time was {idleTime}, so ping request was sent.", nameof(PingServer));
```

Comment:
This is unrelated to the PR, but will help in debugging mqtt logs

Reply:
Does this logging framework have lower log levels? This seems like a trace level log, if the framework supports it.

Reply:
The `Info` event is something that we use to log additional information: https://github.com/Azure/azure-iot-sdk-csharp/blob/abmisr/removeEventHub/common/src/Logging.Common.cs#L195. 
Afaik EventSource doesn't have defined log lovels (like log4j does), but we could always add an additional event called `Trace` and use that for more granular logging.